orion/sites/admin/src/home/home.styl

```diff
@@ -3,4 +3,36 @@
 #view-home
   width: 100%
   height: 400px
-  background-color: colorGrayLight
+  background-color: colorWhite
+
+  & div
+    display: inline-block
+    float:left
+
+#view-home #view-content
+    display: inline-block
+
+  & div
```

Comment:
all the page headers look weird because of this part right here. It's indented incorrectly and somehow stylus made ALL `<div>`s inline-block all around the page which screwed up a lot of stuff. 

Make sure the `&` lines up with the display:inline-block above.